devicetypes/smartthings/zigbee-metering-plug-power-consumption-report.src/zigbee-metering-plug-power-consumption-report.groovy

```diff
@@ -0,0 +1,185 @@
+/**
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition (name: "Zigbee Metering Plug Power Consumption Report", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.smartplug", mnmn: "SmartThings",  vid: "generic-switch-power-energy") {
+        capability "Energy Meter"
+        capability "Power Meter"
+        capability "Actuator"
+        capability "Switch"
+        capability "Refresh"
+        capability "Health Check"
+        capability "Sensor"
+        capability "Configuration"
+        capability "Power Consumption Report"
+
+        fingerprint manufacturer: "DAWON_DNS", model: "PM-B430-ZB", deviceJoinName: "Dawon Outlet" // DAWON DNS Smart Plug PM-B430-ZB (10A), raw description: 01 0104 0051 01 07 0000, 0004, 0003, 0006, 0019, 0702, 0B04 07 0000, 0004, 0003, 0006, 0019, 0702, 0B04
```

Comment:
Please remove all of those fingerprints from the previous DTH (zigbee-metering-plug.groovy) and commit that change in this pull request.

Reply:
Should I reopen the previous pull request(#49017) and delete the fingerprint on that DTH?
Or should I create a new full request?

Reply:
You may change that file and commit to this branch / pull request.

Reply:
I opened the pull request(#50165).

Reply:
Please add those changes (#50165) here, in this pull request.

Reply:
I added it here and closed the PR(#50165).